### PR TITLE
Update n-makefile. 

### PR DESCRIPTION
**Pull request by i-like-robots**
- Use n-makefile and webpack to build client side assets
- Consume client side components via n-ui.
- Add Preact as a dependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,17 +11,38 @@
     "@financial-times/n-logger": "^5.3.0",
     "forever": "^0.15.1",
     "lodash": "^4.8.2",
-    "next-looking-for": "^2.0.1"
+    "next-looking-for": "^2.0.1",
+    "react": "^15.0.2",
```
Comment (wheresrhys) [inline package.json]: preact-compat FTW

Comment (i-like-robots) [inline package.json]: ~~react~~ ++preact

**Comment by wheresrhys** [comment]
Needs to use bower_compoment/n-ui/layout as the server side layouts directory

**Comment by i-like-robots** [comment]
Ahh, thaks. Done and done.

**Comment by wheresrhys** [comment]
:+1:

**Pull request merged**